test/rekt/features/broker/source_to_sink.go

```diff
@@ -21,8 +21,9 @@ import (
 	"fmt"
 
 	"github.com/google/uuid"
-
 	"knative.dev/eventing/test/rekt/resources/broker"
+	"knative.dev/eventing/test/rekt/resources/eventcache"
```

Comment:
I don't really like the cache in the name. not sure what a better name would be though.